message_definitions/v1.0/common.xml

```diff
@@ -2884,6 +2884,79 @@
         <description>Use precision landing, searching for beacon if not found when land command accepted (land normally if beacon cannot be found).</description>
       </entry>
     </enum>
+    <enum name="OPENDRONEID_ID_TYPE">
+      <entry value="0" name="ODID_IDTYPE_NONE">
+        <description>No type defined.</description>
+      </entry>
+      <entry value="1" name="ODID_IDTYPE_SERIAL_NUMBER">
+        <description>Serial Number (ANSI/CTA-2063).</description>
+      </entry>
+      <entry value="2" name="ODID_IDTYPE_CAA_ASSIGNED_ID">
+        <description>CAA (Civil Aviation Authority) Assigned ID.</description>
+      </entry>
+      <entry value="3" name="ODID_IDTYPE_UTM_ASSIGNED_ID">
+        <description>UTM (Unmanned Traffic Management) Assigned ID (UUID RFC4122).</description>
+      </entry>
+    </enum>
+    <enum name="OPENDRONEID_UAS_TYPE">
+      <entry value="0" name="ODID_UASTYPE_NONE">
+        <description>No UAS (Unmanned Air/Aircraft System) type defined.</description>
+      </entry>
+      <entry value="1" name="ODID_UASTYPE_FIXED_WING_POWERED">
+        <description>Fixed Wing Powered.</description>
+      </entry>
+      <entry value="2" name="ODID_UASTYPE_ROTORCRAFT_MULTIROTOR">
+        <description>Rotorcraft/Multirotor.</description>
+      </entry>
+      <entry value="3" name="ODID_UASTYPE_LTA_POWERED">
+        <description>LTA (Lighter than Air, such as a Blimp) Powered.</description>
+      </entry>
+      <entry value="4" name="ODID_UASTYPE_LTA_UNPOWERED">
+        <description>LTA Unpowered (Balloon).</description>
+      </entry>
+      <entry value="5" name="ODID_UASTYPE_VTOL">
+        <description>VTOL (Vertical Take-Off and Landing). Fixed wing aircraft that can take off vertically.</description>
+      </entry>
+      <entry value="6" name="ODID_UASTYPE_FREE_FALL">
+        <description>Free Fall/Parachute.</description>
+      </entry>
+      <entry value="7" name="ODID_UASTYPE_ROCKET">
+        <description>Rocket.</description>
+      </entry>
+      <entry value="8" name="ODID_UASTYPE_GLIDER">
+        <description>Glider.</description>
+      </entry>
+      <entry value="9" name="ODID_UASTYPE_OTHER">
+        <description>Other type of aircraft not .</description>
+      </entry>
+    </enum>
+    <enum name="OPENDRONEID_STATUS_TYPE">
+      <entry value="0" name="ODID_STATUS_UNDECLARED">
+        <description>The status of the UAS is undefined.</description>
+      </entry>
+      <entry value="1" name="ODID_STATUS_GROUND">
+        <description>The UAS is one the ground.</description>
+      </entry>
+      <entry value="2" name="ODID_STATUS_AIRBORNE">
+        <description>The UAS is in the air.</description>
+      </entry>
+    </enum>
+      <enum name="OPENDRONEID_HEIGHT_REFERENCE">
```

Comment:
spacing issue.  I think this file has been strictly formatted using a tool - perhaps doing the same would be a good idea on this PR.

Reply:
Thanks. Good catch. Fixed.
Sorry, this is the first time I contribute to the Mavlink system and I am not 100% familiar with all the process steps. I ran the format_xml.sh script now.